CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- **CUMULUS-2847**
```

Comment:
I'm a bit confused on whether this should be `2847` or `2848` 😅

Reply:
So..... the effort to port it is being tracked in 2848, the original work is 2847.